routes/index.js

```diff
@@ -9,12 +9,16 @@ module.exports = (app) => {
 
   // static paths
   app.get('/', async (req, res) => {
-    res.status(200).render('static/home', {
-      title: 'Home',
-      authenticated: req.session.user ? true : false,
-      message: req.flash('success'),
-      sessionMessage: res.locals.sessionFlash,
-    });
+    let user = req.session.user;
+    if (user) {
+      return res.redirect(`/users/${user._id}/dashboard`);
```

Comment:
redirect to user dashboard if the user is logged in and tries to go to `/` through browser searchbar.